### PR TITLE
Updated MM video example 

### PR DESCRIPTION
**Pull request by raghavdixit99**
# Description

Bugfix due to refactor , added a few packages  @hatianzhang @ravi03071991 

Fixes # (issue)

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- [ ] Added new notebook (that tests end-to-end)

# Suggested Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added Google Colab support for the newly added notebooks.
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I ran `make format; make lint` to appease the lint gods

### CODE REVIEW

File: docs/examples/multi_modal/multi_modal_video_RAG.ipynb [hunk 1]
```diff
@@ -39,7 +39,8 @@
    "outputs": [],
```
Comment (hatianzhang) [inline docs/examples/multi_modal/multi_modal_video_RAG.ipynb:39]: <pre class="ql-syntax" spellcheck="false">Line #7.    context_str = "".join(txt)
</pre><p>could you keep the outputs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/10722/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (raghavdixit99) [reply docs/examples/multi_modal/multi_modal_video_RAG.ipynb:39]: yeah I just saw that nb clean removed outputs , adding them back

Comment (raghavdixit99) [reply docs/examples/multi_modal/multi_modal_video_RAG.ipynb:39]: @hatianzhang resolved

**Comment by raghavdixit99** [comment]
@hatianzhang the pr didnt auto merge  I guess, can you please have a quick look

**Review by hatianzhang — APPROVED** [review]
lgtm

**Pull request merged**